### PR TITLE
Issue #264: Toast notifications (Frontend)

### LINKED ISSUE #264: FRONTEND: Toast Notification
Utilize a Toast library for generating toast notifications. Some Toast examples shown on pages 33 and 63 of the design mockup

### PR DESCRIPTION
**Pull request by yhafez**
### What issue is this solving?
Please connect this Pull Request to the issue it is resolving by using the controls on the right side of this PR.
Resolves #264

### Any helpful knowledge/context for the reviewer?
Is a `npm install` necessary? Yes, I've added the "react-toastify" package
Any special requirements to test? I haven't run into issues on my machine, but just play around with various submission forms throughout the website, and confirm that notifications pop-up as toast notification in the bottom right

This has NOT been tested for mobile. Will likely need some tweaking to look good.

### Please make sure you've attempted to meet the following coding standards
- [x] Code builds successfully
- [x] Code has been tested and does not produce errors
- [x] Code is readable and formatted
- [x] There isn't any unnecessary commented-out code
- [x] There aren't any unnecessary commits or files changed (shouldn't touch the /stashed folder unless the issue requests it)

If you're having trouble meeting this criteria, feel free to reach out on Slack for help!

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -189,11 +192,24 @@ export class App extends React.Component {
                   <PrivateRoute exact path='/request-access/:id' component={RequestAccess} />
                 </div>
               </Switch>
-            {this.state.userSession.isAuthenticated
-              && <Footer />}
-          </div>
-        </BrowserRouter>
-      </UserContext.Provider>
+                {this.state.userSession.isAuthenticated
```
Comment (yhafez) [inline src/App.js]: Ah, so aside from adding the "ToastContainer" here, I cleaned up the indentation which is why there's some changes here.

File: contributing.md [hunk 2]
```diff
@@ -54,6 +54,12 @@ This document provides some overall guidelines and suggestions for how to get st
 
 > We appreciate any and all help with this Volunteer based project, and may check in on issues that are assigned to individuals that have not been updated in some time.
 
+## Adding user notifications
```
Comment (Chris-Boe) [inline contributing.md]: I think we can move this section to the new common components doc!

**Review by Chris-Boe — CHANGES_REQUESTED** [review]
This looks great! We might want to verify the bottom-right placement with Michelle because it seems a bit too out of the way but I love the look and the animation!
Just the one comment on the documentation and some merge conflicts

File: src/App.js [hunk 3]
```diff
@@ -189,11 +192,24 @@ export class App extends React.Component {
...
+                && <Footer />}
+            </div>
+          </BrowserRouter>
+          <ToastContainer
+            position="bottom-right"
```
Comment (yhafez) [inline src/App.js]: @Chris-Boe, if we decide to change the position from the bottom-right, it's as easy as  changing this attribute right here!

**Review by Chris-Boe — APPROVED** [review]

**Pull request merged**